### ISSUE TITLE
keras에서 Sequential 모델 seed 고정 이슈 질문드립니다.

### ISSUE THREAD

**Issue by Joonyoung-Song** [kaggler-tv/dku-kaggle-class#32]
```
import tensorflow as tf
tf.random.set_seed(seed)
np.random.seed(seed)
```

```
def get_model():    
     with tf.device('/device:XLA_GPU:0'):  
        model = Sequential([
            Embedding(vocab_size, embedding_dim, input_length=max_length),
            Bidirectional(LSTM(40, return_sequences=True)),
            # Bidirectional(LSTM(128, return_sequences=True)),
            Bidirectional(LSTM(40)),
            Dense(n_class, activation='softmax')
        ])

        model.compile(loss='categorical_crossentropy', optimizer=Adam(learning_rate=.01))
        return model

```
```

p_val = np.zeros((trn.shape[0], n_class))
p_tst = np.zeros((tst.shape[0], n_class))
for i, (i_trn, i_val) in enumerate(cv.split(trn, y), 1):
    print(f'training model for CV #{i}')
    clf = get_model()
    
    es = EarlyStopping(monitor='val_loss', min_delta=0.001, patience=2,
                       verbose=1, mode='min', baseline=None, restore_best_weights=True)
    
    # rlr = ReduceLROnPlateau(monitor='val_loss', factor=0.5,
    #                     patience=3, min_lr=1e-6, mode='min', verbose=1)

    clf.fit(trn[i_trn], 
            to_categorical(y[i_trn]),
            validation_data=(trn[i_val], to_categorical(y[i_val])),
            epochs=10,
            callbacks=[es
                      #  ,rlr
                       ]
            )
    p_val[i_val, :] = clf.predict(trn[i_val])
    p_tst += clf.predict(tst) / n_fold

```

상기 코드 가장 상단에 seed를 고정했는데 계속 다른 결과가 나옵니다.
동일한 결과를 얻기 위해서는 어떻게 해야 하나요?

(환경은 colab에서 gpu를 사용했습니다.)
